### PR TITLE
#3093 add tests for zookeeper, broker, worker and configurator

### PR DESCRIPTION
**Pull request by saivirtue**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #3093)
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

**Comment by chia7712** [comment]
@saivirtue could you also remove the deprecated route in InfoRoute? You are absolutely capable to modify the backend code :)

File: ohara-manager/client/cypress/api/info.test.js [hunk 1]
```diff
@@ -14,22 +14,65 @@
...
 
+/* eslint-disable no-unused-expressions */
+
+import * as infoApi from '../../src/api/infoApi';
+
 describe('Info API', () => {
-  it('fetchInfo', () => {
-    cy.fetchInfo().then(response => {
+  it('fetchConfiguratorInfo', async () => {
+    await infoApi.fetchConfiguratorInfo().then(response => {
```
Comment (konekoya) [inline ohara-manager/client/cypress/api/info.test.js]: I'm not sure if `Async Await` works in Cypress or not, but we should choose only one pattern and still to it for the sake of consistency: So in this case, use either `Async Await` (if it works okay here) or `Promise` :)

File: ohara-manager/client/cypress/api/info.test.js [hunk 2]
```diff
@@ -14,22 +14,65 @@
...
         data: { isSuccess, result },
       } = response;
 
       const { mode, versionInfo } = result;
-      const { date, revision, version } = versionInfo;
+      const { branch, version, user, revision, date } = versionInfo;
 
       expect(isSuccess).to.eq(true);
 
+      // we using fake configurator to test ; the value should be "FAKE"
```
Comment (konekoya) [inline ohara-manager/client/cypress/api/info.test.js]:
```suggestion
      // we're using fake configurator in our tests; so this value should be "FAKE"
```

File: ohara-manager/client/cypress/api/info.test.js (hunk 1 above)
Comment (saivirtue) [reply ohara-manager/client/cypress/api/info.test.js]: OK

File: ohara-manager/client/cypress/api/info.test.js [hunk 3]
```diff
@@ -14,22 +14,65 @@
...
+      expect(imageName).to.be.a('string');
+      expect(imageName).to.include(serviceName);
+
+      expect(definitions).to.be.an('array');
+      expect(definitions.length > 0).to.be.true;
+
+      const objectKeyDefinitions = definitions
+        .map(definition => definition.key)
+        .filter(key => key === 'group' || key === 'name');
+      expect(objectKeyDefinitions.length).to.eq(2);
```
Comment (konekoya) [inline ohara-manager/client/cypress/api/info.test.js]: Looks like you're explicitly testing these two keys here?

Comment (saivirtue) [reply ohara-manager/client/cypress/api/info.test.js]: Yep.
We cannot assert other "definitions" here, but each service should have it's own `objectKey`, i.e, {name, group}

**Comment by saivirtue** [comment]
retry

File: ohara-manager/client/cypress/api/info.test.js (hunk 3 above)
Comment (konekoya) [reply ohara-manager/client/cypress/api/info.test.js]: As per our discussion, we should fetch the defs first, then add some fake data and save these configs in the backend and assert the response :)

**Comment by saivirtue** [comment]
retry manager-api

**Comment by chia7712** [comment]
retry manager-api

**Comment by konekoya** [comment]
retry it/sshConnector

**Comment by konekoya** [comment]
retry sshConnector

**Pull request merged**